resources/views/frontend/trainee/trainee-profile.blade.php

```diff
@@ -5,51 +5,84 @@
 @extends($layout)
 
 @section('title')
-{{$siteSettingInfo->site_title}} :: {{__('generic.youth_profile')}}
+    {{ $siteSettingInfo->site_title }} :: {{ __('generic.youth_profile') }}
 @endsection
 
 @push('css')
     <style>
         .profile-info-p {
             line-height: normal;
         }
-    </style>
 
+        .custom-button-style {
+            background-color: #671688;
+            border-radius: 12px;
+        }
+
+        .custom-button-style:hover {
+            background-color: #ffffff;
+            color: black;
+            border-color: #671688;
+        }
+
+        @media screen and (max-width: 767px) {
+            .trainee-info {
+                text-align: center;
+                margin-top: 1rem;
+            }
+        }
+
+        .custom-edit-button {
+            border-color: #671688 !important;
+            color: black !important;
+        }
+
+        .custom-edit-button:hover {
+            border-color: #671688 !important;
+            background-color: #671688 !important;
+            color: white !important;
+        }
+    </style>
 @endpush
 
 @section('content')
     <div class="container-fluid">
         <div class="row trainee-profile justify-content-center" id="trainee-profile">
 
-            <div class="col-md-10 mt-2 personal-info-section">
+            <div class="col-md-10 mt-3 pt-1 personal-info-section">
                 <div class="card">
                     <div class="card-header">
                         <div class="card-title">
-                            <h4 class="font-weight-bolder">Personal Information</h4>
+                            <h4>Personal Information</h4>
                         </div>
 
                         <div class="card-tools">
```

Comment:
Try doing this for all the buttons to center the text. 
 <div class="card-tools btn btn-primary d-flex align-items-center justify-content-center btn-rounded custom-button-style px-4 py-2 font-weight-bold">
                            <a href="{{ route('frontend.edit-personal-info') }}"
                                class="">
                                {{ __('admin.common.edit') }}
                            </a>
                        </div>